### ISSUE TITLE
Trying to comment after a failed private profile check can crash the bot

### ISSUE THREAD

**Issue by HerrEurobeat** [HerrEurobeat/steam-comment-service-bot#57]
**Describe the bug**
New skip that doesn't abort the comment process when an error occurred while checking for a private profile seems to be able to crash the bot.

**Full error**
```
[2020-08-19 00:39:10] [Main] comment check for private account error: Error: socket hang up
Trying to comment anyway and hoping no error occurs...
/home/pi/Desktop/steam-comment-service-bot-master/src/bot.js:296
          if (user.privacyState != "public") return respondmethod(403, "Your/the recieving profile seems to be private. Please edit your/the privacy settings on your/the recieving profile and try again!")
                   ^

TypeError: Cannot read property 'privacyState' of undefined
    at community.getSteamUser (/home/pi/Desktop/steam-comment-service-bot-master/src/bot.js:296:20)
    at /home/pi/Desktop/steam-comment-service-bot-master/node_modules/steamcommunity/classes/CSteamUser.js:18:4
    at SteamCommunity._checkHttpError (/home/pi/Desktop/steam-comment-service-bot-master/node_modules/steamcommunity/components/http.js:90:3)
    at Request._callback (/home/pi/Desktop/steam-comment-service-bot-master/node_modules/steamcommunity/components/http.js:50:61)
    at self.callback (/home/pi/Desktop/steam-comment-service-bot-master/node_modules/request/request.js:185:22)
    at Request.emit (events.js:198:13)
    at Request.onRequestError (/home/pi/Desktop/steam-comment-service-bot-master/node_modules/request/request.js:877:8)
    at ClientRequest.emit (events.js:198:13)
    at TLSSocket.socketOnEnd (_http_client.js:435:9)
    at TLSSocket.emit (events.js:203:15)
```

**Comment by HerrEurobeat** [comment]
This issue will stay open until the patch is in the master branch.
The patch will be included in (beta-testing branch) BETA 2.9 b1 and (master branch) 2.9.

**Comment by HerrEurobeat** [comment]
Fixed in commit 5ddef2a (Version 2.9).
